architecture/additional_concepts/storage.adoc

```diff
@@ -192,11 +192,23 @@ For example, NFS can support multiple read/write clients, but a specific NFS PV
 might be exported on the server as read-only. Each PV gets its own set of access
 modes describing that specific PV's capabilities.
 
-Claims are matched to Volumes with similar access modes. A claim's access modes
-represent a request. Therefore, the user may be granted more, but never less.
-For example, if a claim requests RWO, but the only volume available was an NFS PV
-(RWO+ROX+RWX), the claim would match NFS because it supports RWO. Direct matches
-are always attempted first.
+Claims are matched to Volumes with similar access modes. The only two matching
+criteria are access modes and size. A claim's access modes represent a request.
+Therefore, the user may be granted more, but never less. For example, if a claim
+requests RWO, but the only volume available was an NFS PV (RWO+ROX+RWX), the
+claim would match NFS because it supports RWO.
+
+Direct matches are always attempted first. The modes must match or contain more
+modes than you requested. The size must be greater than or equal to what is
+expected. If two types of volumes (NFS and iSCSI, for example) both have the
+same set of access modes, then either of them will match a claim with those
+modes. There is no ordering between types of volumes.
+
+If you have two types of volumes with the same access modes, then here is no
+way to choose one type over another. All volumes with the same modes are
```

Comment:
This sentence and the last sentence of the previous paragraph kind of say the same thing.

Perhaps extend the previous sentence with "There is no ordering between types of volumes and no way to choose one type over another." and then the next paragraph starting with "All volumes with the same modes..."?